Assets/MRTK/Core/Providers/InputSimulation/SimulatedHandDataProvider.cs

```diff
@@ -27,10 +27,7 @@ internal class SimulatedHandState
         // Show a tracked hand device
         public bool IsTracked = false;
         // Activate the pinch gesture
-        public bool IsPinching
-        {
-            get { return gesture == ArticulatedHandPose.GestureId.Pinch; }
-        }
+        public bool IsPinching { get; private set; }
```

Comment:
Could this still be checked whenever `IsPinching` is called instead of setting it as a result of setting `GestureBlending`? Feels like an odd side effect of setting a different property.

```suggestion
        public bool IsPinching => gesture == ArticulatedHandPose.GestureId.Pinch && gestureBlending == 1.0f;
```